src/util/get-github-client-config.ts

```diff
@@ -75,21 +75,20 @@ export async function getGitHubHostname(jiraHost: string, gitHubAppId: number) {
 export async function createAppClient(logger: Logger, jiraHost: string, gitHubAppId: number | undefined): Promise<GitHubAppClient> {
 	const gitHubClientConfig = await getGitHubClientConfigFromAppId(gitHubAppId, jiraHost);
 	return await booleanFlag(BooleanFlags.GHE_SERVER, GHE_SERVER_GLOBAL, jiraHost)
-		? new GitHubAppClient(logger, gitHubClientConfig.baseUrl, gitHubClientConfig.appId.toString(), gitHubClientConfig.privateKey)
-		: new GitHubAppClient(logger);
+		? new GitHubAppClient(gitHubClientConfig, logger, gitHubClientConfig.baseUrl, gitHubClientConfig.appId.toString(), gitHubClientConfig.privateKey)
+		: new GitHubAppClient(gitHubClientConfig, logger);
 }
 
 /**
  * Factory function to create a GitHub client that authenticates as the installation of our GitHub app to get
  * information specific to an organization.
  */
 export async function createInstallationClient(gitHubInstallationId: number, jiraHost: string, logger: Logger, gitHubAppId: number | undefined): Promise<GitHubInstallationClient> {
-
+	const gitHubClientConfig = await getGitHubClientConfigFromAppId(gitHubAppId, jiraHost);
```

Comment:
Maybe we should put the FF here, and pass `null` if FF is off.
Async in side the gitHubClient constructor is kind of weird.

Reply:
Sorry, I'm too lazy to do that. I'd need to do that in 3 places for 3 clients, which means ideally I should abstract this as some helper... given that this code will live a few hours... nah